.github/workflows/sbom.yaml

```
@@ -0,0 +1,32 @@
+name: Generate SBOM
+on:
+  workflow_call:
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+  sbom:
+    needs: build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install Syft
+        run: |
+          curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin
```

@dragomirp after some internal discussions, please replace here "main" with "v0.85.0", it should be safe enough to merge it AS IS, until we have a final SBOM Canonical vision. Tnx!

```suggestion
          curl -sSfL https://raw.githubusercontent.com/anchore/syft/v0.85.0/install.sh | sh -s -- -b /usr/local/bin
```

---

@taurus-forever That doesn't address security concerns—the tag can be moved. Suggest pinning to a commit hash instead

---

@carlcsaposs-canonical I intend to move this to check the image from the registry, so it wouldn't matter if the artefact is potentially changed, but I can't promise it would be this pulse.

---

My point is that `main` and `v0.85.0` are essentially the same from a security perspective—in the short term, I think we should either stay with `main` or move to a commit hash

---

Yes. Running syft at the end against the registry instead of the artefact would address this, since at that point the artefact is already published.

---

> but I can't promise it would be this pulse

For the commit hash, I'm only referring to short-term (i.e. this PR)